examples/server/server.cpp

```diff
@@ -14,7 +14,9 @@
 // mime type for sending response
 #define MIMETYPE_JSON "application/json; charset=utf-8"
 
-// auto generated files (update with ./deps.sh)
+// auto generated files update by:
+// $ npm run build --prefix examples/server/webui (updates examples/server/public/index.html.gz)
+// $ cmake --build build --target llama-server    (generates build/examples/server/index.html.gz.hpp)
```

Comment:
There is a guide already in server/README, so I think we should point to the README instead of copying it here, so in the future we won't forget this again

Reply:
Btw in the README, can you also add the clarification for `cmake` command to generate `index.html.gz.hpp` ?

Reply:
I've moved the comments to the README.md in https://github.com/ggerganov/llama.cpp/pull/11484/commits/6d21abfc201ae0f1fad5fe4991785618e17bc3e5.